README.md

```diff
@@ -6,9 +6,10 @@ This is a mobile app project template using [Uno](https://github.com/unoplatform
 
 We use `dotnet` project templates to easily create new projects. It simplifies the **project renaming** and supports **conditional inclusions**.
 
-1. Install the template using the following command at the root of the project.
+1. Clone this repository on your machine and open a command prompt at the root of the project and run the following command 
```

Comment:
I think this should still mention that it's installing the template on your machine.

Reply:
It might be a good idea to split this doc into 2 parts:
1. Installing and uninstalling the template.
2. Running the template to generate a new project.